pandas/core/generic.py

```diff
@@ -5548,53 +5548,74 @@ def clip(self, lower=None, upper=None, axis=None, inplace=False,
         """
         Trim values at input threshold(s).
 
+        Truncates values below and above specified thresholds.
+        Thresholds can be singular values or array like, and in
+        the latter case the truncation is performed element-wise
+        in the specified axis.
+
         Parameters
         ----------
         lower : float or array_like, default None
+            Minimum threshold value. All values below this
+            threshold will be set to it.
         upper : float or array_like, default None
+            Maximum threshold value. All values above this
+            threshold will be set to it.
         axis : int or string axis name, optional
             Align object with lower and upper along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
+        args : iterable, optional
+            Arguments to pass to the function.
+        kwargs : mapping, optional
+            Keyworded arguments to pass to the function.
```

Comment:
See discussion on https://github.com/pandas-dev/pandas/pull/19985/files#diff-248241e14f081931cc805f9d0e137a5fR709

maybe change to:
```
*args, **kwargs 
    Additional keywords have no effect but might be accepted 
    for compatibility with numpy.
```